charts/db-operator/templates/_helpers.tpl

```diff
@@ -25,65 +25,95 @@ If release name contains chart name it will be used as a full name.
+{{- else -}}
+{{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
 {{- end -}}
 
 {{/*
-Image version definition;
+Arguments builder
 */}}
-{{- define "db-operator.image_version" -}}
-{{ default .Chart.AppVersion .Values.image.tag }}
+{{- define "db-operator.args" -}}
+{{- $args := list -}}
+{{- if .Values.checkForChanges -}}
+{{- $args = append $args "--check-for-changes" -}}
+{{- end -}}
+{{ join "," $args }}
 {{- end -}}
 
 {{/*
-Image version definition using Github Packages format ('v' prefix);
+Webhook templates
 */}}
-{{- define "db-operator.github_packages_image_version" -}}
-{{- printf "v%s" (default .Chart.AppVersion .Values.image.tag) }}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "webhook.name" -}}
+{{- printf "webhook" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "webhook.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
 {{- end -}}
 
 {{/*
 Create the name of the service account to use
 */}}
-{{- define "db-operator.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create -}}
-{{- printf "%s-sa" (include "db-operator.name" .) -}}
+{{- define "webhook.serviceAccountName" -}}
+{{- if .Values.webhook.serviceAccount.create -}}
+    {{ default (include "webhook.fullname" .) .Values.webhook.serviceAccount.name }}
 {{- else -}}
-{{ default "default" .Values.serviceAccount.name }}
+    {{ default "default" .Values.webhook.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
 
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
 {{/*
 Common labels
 */}}
-{{- define "db-operator.labels" -}}
-helm.sh/chart: {{ include "db-operator.chart" . }}
-{{ include "db-operator.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
+{{- define "labels" -}}
+helm.sh/chart: {{ include "chart" . }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
+{{- if eq (default "helm" .Values.creator) "helm" }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
 {{- end -}}
+{{- end -}}
 
 {{/*
-Selector labels
+Image version definition;
 */}}
-{{- define "db-operator.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "db-operator.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
+{{- define "image_version" -}}
+{{ default .Chart.AppVersion .Values.image.tag }}
```

Comment:
Isn't it going to be a breaking change? I wanted to get rid of this "v" prefix, but I guess it will break all the installations. 
Also, we have the versions without the prefix in the `Chart.yaml` and out images are pushed with v, so it should not work at all, I guess. 

I'd like to make do this change later,

Reply:
I had a look at it and it's actually simply existing code which I moved to the end of the file. The only differences are that I renamed the definitions from `db-operator.image_version` and `db-operator.github_packages_image_version` to `image_version` and `github_packages_image_version`. That hopefully shouldn't cause any changes at all.

Reply:
Sorry, I've missed that part: 
```
{{- printf "v%s" (default .Chart.AppVersion .Values.image.tag) }}

```